ui/src/CheckSources.js

```diff
@@ -35,22 +40,22 @@ const CheckSources = React.createClass({
   getInitialState() {
     return {
       isFetching: true,
-      sources: [],
     };
   },
 
   componentDidMount() {
     getSources().then(({data: {sources}}) => {
-      this.setState({sources, isFetching: false});
+      this.props.loadSourcesAction(sources);
```

Comment:
Was thinking about doing that, but wasn't sure if it was best practices in redux.